external/storm-hdfs/src/main/java/org/apache/storm/hdfs/bolt/HdfsBolt.java

```diff
@@ -80,6 +86,11 @@ public HdfsBolt addRotationAction(RotationAction action){
         return this;
     }
 
+    public HdfsBolt withTickTupleIntervalSeconds(int interval) {
```

Comment:
Assuming that would address default, min and max values for the interval, should be fine.